glide.yaml

```diff
@@ -36,16 +36,16 @@ import:
   version: 5c3c0fce48842b2c0bbaa99b4e61b0175d84b47c
 
 - package: github.com/uber-go/tally
-  version: v1.1.0
+  version: v3.0.0
```

Comment:
Perhaps we make this a bit looser with "<4" so that any possible compatible library can be used.